core.ts

```diff
@@ -1325,7 +1345,7 @@ export declare namespace PinejsClientCoreFactory {
 		| '$cast';
 	type FilterFunctionValue = Filter;
 
-	export interface FilterObj extends Dictionary<Filter | undefined> {
+	export interface FilterObj extends Dictionary<Filter | Lambda | undefined> {
```

Comment:
Is that indeed valid? Can I do `{ something: { $alias, 'a', $expr: ... } }`?
I thought we only could do that in `$any` & `$all`.

Reply:
No you can't, but the dictionary type has to cover every possible key for if you do eg
```ts
const x: string = '$any'
filterObj[x] // Lambda is the only actually valid option but because it's `string` it means that as far as typescript is concerned it could be the value of any string key